tests/http/test_http_parser.py

```diff
@@ -23,6 +23,13 @@ class TestHttpParser(unittest.TestCase):
     def setUp(self) -> None:
         self.parser = HttpParser(httpParserTypes.REQUEST_PARSER)
 
+    def test_issue_127(self) -> None:
+        self.parser.parse(CRLF)
+        self.assertEqual(self.parser.state, httpParserStates.INITIALIZED)
+        raw = b'qwqrqw!@!#@!#ad adfad\r\n'
```

Comment:
Thank you for this test case.  This prompted me to post previous comment i.e. raise exception instead of ignoring.

Because if in this test we add the following:

```python
while 1:
    self.parser.parse(b'qwqrqw!@!#@!#ad adfad\r\n')
```

server will keep on spinning (because it will ignore invalid bytes), resulting in lower performance for other clients connecting to the server